sources/sqlserver/data.go

```diff
@@ -57,10 +57,7 @@ func ConvertData(conv *internal.Conv, srcTable string, srcCols []string, srcSche
 	}
 	for i, spCol := range spCols {
 		srcCol := srcCols[i]
-		// Skip columns with 'NULL' values. When processing data rows from mysqldump, these values
-		// are represented as nil (by pingcap/tidb/types/parser_driver's ValueExpr), which is
-		// converted to the string '<nil>'. When processing data rows obtained from the MySQL driver,
-		// 'NULL' values are represented as "NULL" (because we retrieve the values as strings).
+		// Skip columns with 'NULL' values.
 		if vals[i] == "<nil>" || vals[i] == "NULL" {
```

Comment:
We should remove the vals[i] == "<nil>" condition.

Reply:
You are right, removed.